sdk/tables/azure-data-tables/azure/data/tables/_table_client.py

```diff
@@ -605,25 +511,16 @@ def list_entities(
         results_per_page: Optional[int] = None,
         select: Optional[Union[str, List[str]]] = None,
         **kwargs,
-    ) -> ItemPaged[TableEntity]:
+    ) -> ItemPaged[EntityType]:
```

Comment:
I think changing this is going to cause mypy errors for customers - I think perhaps if `flatten_entity...` is True, then maybe we should keep returning `TableEntity` but `metadata = {}` or something....